apps/backoffice-v2/src/pages/Entity/components/Case/Case.Documents.Toolbar.tsx

```diff
@@ -31,6 +38,13 @@ export const DocumentsToolbar: FunctionComponent<{
 
   return (
     <div className={`absolute z-50 flex space-x-2 bottom-right-6`}>
+      {image && (
```

Comment:
Any reason the condition is not the same as the button below? The isLoading and image id.